### ISSUE TITLE
Question regarding details in the paper

### ISSUE THREAD

**Issue by minhquoc0712** [Saehyung-Lee/DCC#1]
Thank you for your interesting research.

I have some questions regarding some equations in the paper, and I would be very happy if you could explain them to me.

1. What is the meaning and motivation of the first term on the right-hand side in equation (4)
2. In equation (10), in the weights update rule, why do you divide the gradient by |S|? I thought the normalized factor was already included in the loss function.

**Comment by Saehyung-Lee** [comment]
Thank you for your interest in our work.
1. Without the constraint term in equation (4), the synthetic data can be placed anywhere on the feature space. Therefore, we cannot compare the dataset condensation methods without the constraint term because the sample complexity of our motivating example is 1 sample/class.
2. In equation (10) L is the cross-entropy loss function.

**Comment by minhquoc0712** [comment]
Thank you for your response.
